pkg/services/container/announcement/load/storage/storage.go

```diff
@@ -40,21 +40,28 @@ type storageKey struct {
 type Storage struct {
 	mtx sync.RWMutex
 
-	mItems map[storageKey]*usedSpaceEstimations
+	estLifeCycle uint64
+	mItems       map[storageKey]*usedSpaceEstimations
 }
 
 // Prm groups the required parameters of the Storage's constructor.
 //
 // The component is not parameterizable at the moment.
-type Prm struct{}
+type Prm struct {
+	// EstimationsLifeCycle is a number (in epochs) estimations
+	// are kept in the [Storage] instance. Note, current epoch
+	// is controlled with [Storage.NewEpoch].
+	EstimationsLifeCycle uint64
+}
 
 // New creates a new instance of the Storage.
 //
 // The created Storage does not require additional
 // initialization and is completely ready for work.
-func New(_ Prm) *Storage {
+func New(prm Prm) *Storage {
```

Comment:
`New(maxAge uint64) *Storage`? What exactly `Prm` is solving for us here?

Reply:
i am not a dev who done that so did not want to be a dev who changed it. but do not mind, do not like `Prm` approach, fixed

Reply:
It was completely unused, so a pretty easy choice here.